docs/builtins/Repl/typecheck.md

```diff
@@ -1,52 +1,54 @@
 ## typecheck
 
-Use `(typecheck "my-module")` to run the pact static typechecker on some module `my-module`
+Use `typecheck` to run the Pact static type checker on a specified `module` name.
 
 ### Basic syntax
 
-For some module `foo`:
+To run the static type checker on a specified `module` name, use the following syntax:
```

Comment:
Like I said above, we're not running it on the module name, the typechecker runs on the module pointed to by the first parameter.